### PR TITLE
Problem: mac release minos is 11.0 (fix #168)

### LINKED ISSUE #168: Problem: mac release minos is 11.0
lower minos to 10.15

### PR DESCRIPTION
**Pull request by leejw51crypto**
Solution: change os to 10.15

### CODE REVIEW

**Comment by elecming** [comment]
if `checkmac.sh` is not working, how about removing it?

**Review by tomtau — APPROVED** [review]

File: .github/workflows/mac-build.yml [hunk 1]
```diff
@@ -15,18 +15,14 @@ on:
...
   mac-build:
     runs-on: macos-11
+    env: 
+          MACOSX_DEPLOYMENT_TARGET: 10.15        
     steps:
       - uses: actions/checkout@v2
         with:
-          submodules: recursive
-
+          submodules: recursive
```
Comment (elecming) [inline .github/workflows/mac-build.yml]: redundant tail spaces

File: .github/workflows/mac-build.yml [hunk 2]
```diff
@@ -15,18 +15,14 @@ on:
...
-
+          submodules: recursive          
       - name: Build play-cpp-sdk library
-        run: ./checkmac.sh && cargo build --package play-cpp-sdk --release
-
-      - name: Build demo project
-        working-directory: demo
-        run: make
-
+        run: make build_cpp
```
Comment (elecming) [inline .github/workflows/mac-build.yml]: maybe just `make` which is simpler.

Comment (leejw51crypto) [reply .github/workflows/mac-build.yml]: ok

File: .github/workflows/mac-build.yml (hunk 1 above)
Comment (leejw51crypto) [reply .github/workflows/mac-build.yml]: ok

**Review by elecming — APPROVED** [review]

**Comment by leejw51crypto** [comment]
after checking ci test, i'll merge

**Pull request merged**